pkg/model/nlb/builder.go

```diff
@@ -0,0 +1,314 @@
+package nlb
+
```

Comment:
i prefer the builder to be not inside model package.(which is only for model)

how about `/pkg/service/nlb_model_builder.go` or `/pkg/service/nlb/model_builder.go`